ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.test.tsx

```diff
@@ -90,6 +98,40 @@ describe('ScrollToBottom', () => {
       });
     });
 
+    it('does not scroll to top when the confirmation changes', () => {
```

Comment:
should the test be `'does not scroll to top when the confirmation id does not change'`?

Reply:
ty! updated [533bbd7](https://github.com/MetaMask/metamask-extension/pull/24435/commits/533bbd79a1a7bfe24fa7f15cd894b8905a348165)